terraform/modules/helm/main.tf

```
@@ -156,6 +156,14 @@ resource "kubectl_manifest" "selfsigned_issuer" {
   yaml_body  = file("${path.module}/clusterissuer-selfsigned.yaml")
 }
 
+locals {
+  jsonpatch = jsonencode([{
```

```suggestion
  local_exec_quotes = startswith(abspath(path.module), "/") ? "'" : ""
  jsonpatch = jsonencode([{
```

---

Yes it works on windows. But this function kind of obfuscates the logic. I would leave a comment for the next ones reading it.
```suggestion
  # local_exec_quotes is a helper function to deal with different handling of
  # quotes between linux and windows. On linux, it will output "'". On windows,
  # it will output "".
  local_exec_quotes = startswith(abspath(path.module), "/") ? "'" : ""
  jsonpatch = jsonencode([{
```